vf_sv/site_core.php

```diff
@@ -651,6 +651,7 @@
 $Definition['Failed to find discussion for commenting.'] = 'Hittade ingen diskussion att kommentera.';
 $Definition['Failed to load the poll.'] = 'Misslyckades att ladda omröstning.';
 $Definition['Favorites'] = 'Favoriter';
+$Definition['Featured Categories'] = 'Featured Categories';
```

Comment:
This is in the wrong file (should be in `tx_source`). You've put it in with some unknown language.

Reply:
Yeah thanks, I've added to the correct site_core instead.